lib/rails_admin/config/fields/types/timestamptz.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# :nocov:
```

Comment:
I think the :nocov: labels say that this doesn't quite work. I think it needs explicitly requiring somehow.

Reply:
How do the :nocov: labels mean that? 

I did wonder about the explicit requiring. There is another file I can add which does require the individual field types in Rails Admin - one person who had implemented this fix said it wasn't needed to work on their end....but perhaps I'll add it